src/workerd/io/worker.c++

```diff
@@ -1253,20 +1253,11 @@ Worker::Worker(kj::Own<const Script> scriptParam,
     }
   });
 
-  auto maybeMakeSpan = [&](auto operationName) -> MaybeSpan {
-    if (auto span = systemTracer.makeSpan(operationName)) {
-      span.setTag("truncated_script_id"_kj, truncateScriptId(script->getId()));
-      return kj::mv(span);
-    } else {
-      return MaybeSpan();
-    }
-  };
-
-  MaybeSpan currentSpan = maybeMakeSpan("lw:new_startup_metrics"_kj);
+  auto currentSpan = SpanBuilder(parentSpan, "lw:new_startup_metrics"_kj);
```

Comment:
Kinda liked the makeSpan or makeSubspan helper personally. Reads better in my mind

```c++
auto currentSpan = parentSpan.makeSpan("lw:new_startup_metrics"_kj);
// vs
auto currentSpan = SpanBuilder(parentSpan, "lw:new_startup_metrics"_kj);
```

Reply:
Yeah actually I agree, much better for auto-complete that way.